tests/research/test_method_selection.py

```
@@ -0,0 +1,310 @@
+
+    assert selection_dates == dates[:-252]
+    assert final_dates == dates[-252:]
+    assert captured["market_dates"] == selection_dates
+    assert captured["counterfactual_feature_sets"] == feature_names
+    assert captured["complete_case_extra_columns"] == (
+        "open_to_open_volatility_20", "open_to_open_volatility_60", "open_to_open_volatility_252"
+    )
+
+
+def test_create_matrix_connects_phase_a_phase_b_and_evidence(monkeypatch):
+    feature_sets, specs_by_id = service.build_feature_set_manifests()
```

Ruff (0.16.2)

[warning] Unpacked variable `specs_by_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)